enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1222 @@
+- `RRULE:FREQ=YEARLY` - An event that occurs once a year on a specific date. 
+- `RRULE:FREQ=WEEKLY;INTERVAL=2` - An event that occurs every two weeks.
+
+All such expressions shall be evaluated with a starting date of Jan 1st, 1970 00:00Z. In other
+words, `RRULE:FREQ=YEARLY` would be considered permissive, for one day, at the start of each new year.
+
+If no `startTime` or `duration` is specified, any day selected by the RRULE will suggest a
+permissive 24h window unless a date is in the `exclude` ranges.
+
+**RRULE Constraints**
+A valid RRULE for change management:
+- must identify a date, so, although RRULE supports `FREQ=HOURLY`, it will be rejected if an attempt it made to use it.
+- cannot specify an end for the pattern. `RRULE:FREQ=DAILY;COUNT=3` suggests
+  an event that occurs every day for three days only. As such, neither `COUNT` nor `UNTIL` is 
+  supported and will be rejected if an attempt is made to use them.
+- cannot specify a permissive window more than 2 years away.
+
+**Overview of Interactions**
+The MaintenanceSchedule strategy, along with `changeManagement.pausedUntil` allows a cluster lifecycle administrator to express 
+one of the following:
+
+| pausedUntil    | permit | exclude | Enforcement State (Note that **effective** state must also take into account hierarchy)                                                                                                                                                                                  |
+|----------------|--------|---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `null`/`false` | `null` | `null`  | Permissive indefinitely           |
+| `true`         | *      | *       | Paused indefinitely |
+| `null`/`false` | set    | `null`  | Permissive during reoccurring windows time. Paused at all other times.                                                                                                                  |
+| `null`/`false` | set    | set     | Permissive during reoccurring windows time modulo excluded date ranges during which it is paused. Paused at all other times.                                                                                                                                                                                                         |
+| `null`/`false` | `null` | set     | Permissive except during excluded dates during which it is paused.                                                                                                                             |
+| date           | *      | *       | Honor permit and exclude values, but only after the specified date. For example, permit: `null` and exclude: `null` implies the strategy is indefinitely permissive after the specified date. |
+
+
+#### MachineConfigPool Assisted Strategy Configuration

```

Comment:
Not seeing where a forced drain would be configured.  In a managed context we need have this as an option, if only for our own clusters.  We can decide what default value to use (today it is 1 hour I believe).

Reply:
Forced drain is a separate EP that I hope will also reduce MUO: https://github.com/openshift/enhancements/pull/1616 .